.github/workflows/tests.yml

```diff
@@ -40,13 +40,13 @@ jobs:
         PWNDBG_GITHUB_ACTIONS_TEST_RUN=1 ./tests.sh --cov
 
     - name: Process coverage data
-      if: matrix.os == 'ubuntu-22.04'
+      if: matrix.os == 'ubuntu-22.04' && success() || failure()
```

Comment:
I think we should keep this one the way it was. If something breaks with this step, it's very likely all fault, whereas the next step relies on network connectivity/rate limiting, so if something breaks there it's more likely not our fault and we don't want the job to fail.